batch2/batch/driver/instance_pool.py

```diff
@@ -208,8 +206,8 @@ def add_instance(self, instance):
     -e CORES=$CORES \
     -e NAME=$NAME \
     -e NAMESPACE=$NAMESPACE \
-    -e INST_TOKEN=$INST_TOKEN \
-    -e INTERNAL_IP=$INTERNAL_IP \
+    -e TOKEN=$TOKEN \
+    -e IP_ADDRESS=$IP_ADDRESS \
     -e BUCKET_NAME=$BUCKET_NAME \
     -e INSTANCE_ID=$INSTANCE_ID \
```

Comment:
This `INSTANCE_ID` appears to be only used as a location to store logs. How does this differ from the machine name? Should we eliminate it? It seems confusing that the logs are stored in a location derived from the instance *id* when the *name* is used everywhere.

Reply:
INSTANCE_ID is badly named: it is the running instance of the batch server, used as a prefix for logs so different instance of the batch server (e.g. from different PRs being tested) don't collide with each other.